zerver/models.py

```diff
@@ -606,19 +646,25 @@ def get_stream_backend(stream_name, realm):
         name__iexact=stream_name.strip(), realm_id=realm_id)
 
 def get_active_streams(realm):
+    # type: (Realm) -> QuerySet
+    # TODO: Return_Type_Is_QuerySet_Or_List[stream] because in others objects.filter it
+    # seems to be a list but here it is not as we are using it in line 680
     """
     Return all streams (including invite-only streams) that have not been deactivated.
     """
     return Stream.objects.filter(realm=realm, deactivated=False)
 
 # get_stream takes either a realm id or a realm
 def get_stream(stream_name, realm):
+    # type: (str, Union[int, Realm]) -> Stream
     try:
         return get_stream_backend(stream_name, realm)
     except Stream.DoesNotExist:
         return None
 
 def bulk_get_streams(realm, stream_names):
+    # type: (Realm, STREAM_NAMES) -> Dict[Any, Any]
```

Comment:
I think until then it'll be better to use `TypeVar('STREAM_NAMES', Sequence[str], AbstractSet[str])` instead of `TypeVar('STREAM_NAMES', List[str], Set[str])`